crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -37,6 +39,57 @@ impl DynamicScene {
         Self::from_world(&scene.world, type_registry)
     }
 
+    /// Create a Dynamic Scene with specific entities.
+    ///
+    /// The generic parameter is used as a `Query` filter.
```

Comment:
We should give examples of valid syntax.

Reply:
done!